packages/theme/json/theme.json

```diff
@@ -985,826 +960,640 @@
       },
-      "color-blue-100-fade-fill": {
-        "$value": {
-          "color": "{foundations.color.color-blue-100}",
-          "opacity": "{palette.opacity.palette-opacity-disabled}"
-        },
+      "color-transparent": {
+        "$value": "#ffffff00",
         "$type": "color"
       },
-      "color-blue-600-fade-fill": {
-        "$value": {
-          "color": "{foundations.color.color-blue-600}",
-          "opacity": "{palette.opacity.palette-opacity-disabled}"
-        },
+      "color-violet-400": {
+        "$value": "#8a96f2",
         "$type": "color"
       },
-      "color-white-fade-separatorOpacity-primary": {
-        "$value": {
-          "color": "{foundations.color.color-white}",
-          "opacity": "{palette.opacity.palette-opacity-primary-border}"
-        },
+      "color-violet-500": {
+        "$value": "#636ebf",
         "$type": "color"
       },
-      "color-white-fade-separatorOpacity-secondary": {
-        "$value": {
-          "color": "{foundations.color.color-white}",
-          "opacity": "{palette.opacity.palette-opacity-secondary-border}"
-        },
+      "color-violet-600": {
+        "$value": "#454fa1",
         "$type": "color"
       },
-      "color-white-fade-separatorOpacity-tertiary": {
-        "$value": {
-          "color": "{foundations.color.color-white}",
-          "opacity": "{palette.opacity.palette-opacity-tertiary-border}"
-        },
+      "color-white": {
+        "$value": "#ffffff",
         "$type": "color"
       },
-      "color-black-fade-separatorOpacity-primary": {
-        "$value": {
-          "color": "{foundations.color.color-black}",
-          "opacity": "{palette.opacity.palette-opacity-primary-border}"
-        },
+      "color-white-15a": {
+        "$value": "#ffffff00",
         "$type": "color"
       },
-      "color-black-fade-separatorOpacity-secondary": {
-        "$value": {
-          "color": "{foundations.color.color-black}",
-          "opacity": "{palette.opacity.palette-opacity-secondary-border}"
-        },
+      "color-white-25a": {
+        "$value": "#ffffff00",
         "$type": "color"
       },
-      "color-black-fade-separatorOpacity-tertiary": {
-        "$value": {
-          "color": "{foundations.color.color-black}",
-          "opacity": "{palette.opacity.palette-opacity-tertiary-border}"
-        },
+      "color-white-40a": {
+        "$value": "#ffffff00",
         "$type": "color"
       },
-      "color-black-fade-background-hover": {
-        "$value": {
-          "color": "{foundations.color.color-black}",
-          "opacity": "{foundations.opacity.opacity-8}"
-        },
+      "color-white-70a": {
+        "$value": "#ffffff00",
         "$type": "color"
       },
-      "color-white-fade-background-hover": {
-        "$value": {
-          "color": "{foundations.color.color-white}",
-          "opacity": "{foundations.opacity.opacity-8}"
-        },
+      "color-white-8a": {
+        "$value": "#ffffff00",
```

Comment:
unsure where opacity stands in theme 2.0, if we would link to an opacity foundation here, and have color and opacity as separate values like the JSON structure for theme 1 fade tokens @origami-z